### ISSUE TITLE
Unable to start latest nightly on linux-x64

### ISSUE THREAD

**Issue by menski** [camunda/camunda-modeler#246]
Hi,

I did my daily modeler nightly update and now it isn't starting anymore. Is this a known issue?

Used: https://camunda.org/release/camunda-modeler/nightly/camunda-modeler-linux-x64.tar.gz
Version: v0.7.0-nightly (build 1459899386)

Error:
```
$ ./camunda-modeler 
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module './client-config'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:289:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/home/menski/.local/opt/camunda-modeler/resources/app.asar/app/lib/index.js:23:20)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Object.<anonymous> (/home/menski/.local/opt/camunda-modeler/resources/atom.asar/browser/lib/init.js:158:8)

```

**Comment by nikku** [comment]
Will have a look into it.

**Comment by nikku** [comment]
Should be fixed via 1c466d4ae209b9c6080d987b41a07651b081ccc5
